dlt/common/runtime/collector.py

```diff
@@ -333,19 +383,28 @@ def update(
             bar = alive_bar(total=total, title=desc, **self.alive_kwargs)
             self._bars_contexts[key] = bar
             bar = self._bars[key] = bar.__enter__()
+            self._bars_counts[key] = 0
+        else:
+            # TODO: implement once total change is supported
```

Comment:
alive bars don't seem to support updating the total, you can create a new one and remember the old settings, but then a new bar appears in the ui, and the old half files one stays..

Reply:
alive is just for fun. so OK :)